modules/security-context-constraints-psa-opting.adoc

```diff
@@ -10,22 +10,24 @@ You can enable or disable automatic pod security admission synchronization for m
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
+Most namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
 
 * `default`
 * `kube-node-lease`
 * `kube-system`
 * `kube-public`
 * `openshift`
-* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`
+* System-created namespaces that are prefixed with `openshift-` and do not contain an Operator
```

Comment:
I think it's just "system created(payload created) NSes that are prefixed with openshift-*".  The syncer will *never* sync labels onto those NSes, even if you add an operator to the NS.

see: https://github.com/openshift/cluster-policy-controller/blob/523e4104074f88047de7d725d5c99ca4dbf8c167/pkg/psalabelsyncer/nsexemptions/nsexemptions.go#L9